src/display/process/procGraphs.go

```diff
@@ -30,21 +31,22 @@ import (
 var runProc = true
 
 func getChildProcs(proc *process.Process) []string {
-	childProcs := []string{"PID                   Command"}
+	headerString := "PID" + strings.Repeat(" ", 19) + "Command"
+	childProcs := []string{headerString}
 	for _, proc := range proc.Children {
+		var processData, spacesForCommandRowData string
+		processPid := strconv.Itoa(int(proc.Pid))
+		// 22 reflects position where row data for "Command" column should start (headerString has 19 spaces + length of ("PID") is 3 i.e. 22)
+		spacesForCommandRowData = strings.Repeat(" ", 22-len(processPid))
+		processData = "[" + processPid + "](fg:yellow)" + spacesForCommandRowData
 		exe, err := proc.Exe()
 		if err == nil {
-			temp := strconv.Itoa(int(proc.Pid))
-			for i := 0; i < 22-len(strconv.Itoa(int(proc.Pid))); i++ {
-				temp = temp + " "
-			}
-			temp = temp + "[" + exe + "](fg:green)"
-			childProcs = append(childProcs, temp)
+			processData += "[" + exe + "](fg:green)"
 		} else {
-			childProcs = append(childProcs, "["+strconv.Itoa(int(proc.Pid))+"](fg:yellow)"+"            "+"NA")
+			processData += "NA"
```

Comment:
Hey @siddhant94, Leaving NA white makes this look kind of out of place. Also this colour scheme is kind of out of place from the other table we see when we run `grofer proc`. We're sorry for the inconvenience, but could you get this list to follow the same colour scheme please? 

The colour scheme is something like this:

| Field | Colour |
|--------|----------|
|  pid | white |
| process | green |
| selected row (both pid and process) | yellow |

Reply:
@Gituser143 Sure :+1: 
Thanks for providing the color-scheme. Can you check this screenshot to verify if this is the desired behaviour?
headings -> white, `PID` & `NA` -> white, process -> green & selectedRow -> yellow

![bug-fix-screenshot](https://user-images.githubusercontent.com/8606880/94938535-91d03100-04ee-11eb-95dc-710bdcea9b99.png)

Reply:
This looks good! @MadhavJivrajani do you think NA should be green too?